### PR TITLE
Add quote show endpoint

### PR DESCRIPTION
**Pull request by rhiroyuki**
This PR adds the endpoint to show a single quote on `/quotes/:id`

solves #5

### CODE REVIEW

**Review by marcoscoelho1 — APPROVED** [review]
Approved!

File: app/controllers/quotes-controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const QuotesController = {
+  create: (req, res, next) => {
+    let quote = new Quote(req.body);
+
+    if(req.body.message !== ''){
+      quote.save((err)=>{
+        if(err){
+          console.log(err);
+        } else {
+          console.log("Successfully created a quote")
```
Comment (juliofortunato) [inline app/controllers/quotes-controller.js]: 💅 
```suggestion
          console.log("Successfully created a quote");
```

File: app/controllers/quotes-controller.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          res.status(201).send({ quote }).end()
```
Comment (juliofortunato) [inline app/controllers/quotes-controller.js]: 💅 
```suggestion
          res.status(201).send({ quote }).end();
```

File: app/controllers/quotes-controller.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        return res.status(404).send({ message: 'Invalid ID' }).end();
+      }
+
+      res.send({
+        data: {
+          message: quote.message,
+          author: quote.author
+        }
+      });
+    })
```
Comment (juliofortunato) [inline app/controllers/quotes-controller.js]: 💅 
```suggestion
    });
```

**Review by juliofortunato — CHANGES_REQUESTED** [review]

**Review by juliofortunato — APPROVED** [review]

**Pull request merged**